helm/ph-ee-engine/values.yaml

```diff
@@ -43,6 +43,43 @@ zeebe-cluster-helm:
       value: "2"
     - name: ZEEBE_BROKER_BACKPRESSURE_VEGAS_BETA
       value: "8"
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_EVENT
+      value: 'true'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_VARIABLE
+      value: 'true'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_PROCESSINSTANCE
+      value: 'true'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_PROCESS
+      value: 'true'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_INCIDENT
+      value: 'true'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_COMMAND
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_REJECTION
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_JOB
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_DEPLOYMENT
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_ERROR
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_JOBBATCH
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_JOB
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_MESSAGE
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_MESSAGESUBSCRIPTION
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_VARIABLEDOCUMENT
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_JOB
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_WORKFLOWINSTANCE
+      value: 'false'
+    - name: ZEEBE_BROKER_EXPORTERS_KAFKA_ARGS_INDEX_CREATETEMPLATE
+      value: 'false'
+
```

Comment:
`args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com/jars/exporter-1.0.0-SNAPSHOT.jar"; ls -al /exporters/']`

This needs changing to fynarfin.io/images/jarname.jar-1.1.0 as pipelines aren't integrated with S3.

Reply:
but in aws its still 1.0.0

Reply:
updated PR